README.md

```diff
@@ -28,23 +28,6 @@ public function testCanBeUsedAsString(): void
     );
 }
 ```
-
-You should also have an active item in the project settings at
-
-```
-https://app.qase.io/project/QASE_PROJECT_CODE/settings/options
-```
-
-options in the `Test Runs` block:
-
-```
-Auto create test cases
```

Comment:
I think taht only `Allow submitting results in bulk` setting is removed, isn't it?

Reply:
The first setting was eliminated earlier, and now both